test/VarianceTest.cc

```diff
@@ -341,19 +342,141 @@ INSTANTIATE_TEST_CASE_P(
+        VarianceParam(16, 32, &eb_aom_variance16x32_c,
+                      &eb_aom_variance16x32_avx2),
+        VarianceParam(16, 64, &eb_aom_variance16x64_c,
+                      &eb_aom_variance16x64_avx2),
         VarianceParam(32, 8, &eb_aom_variance32x8_c, &eb_aom_variance32x8_avx2),
-        VarianceParam(32, 16, &eb_aom_variance32x16_c, &eb_aom_variance32x16_avx2),
-        VarianceParam(32, 32, &eb_aom_variance32x32_c, &eb_aom_variance32x32_avx2),
-        VarianceParam(32, 64, &eb_aom_variance32x64_c, &eb_aom_variance32x64_avx2),
-        VarianceParam(64, 16, &eb_aom_variance64x16_c, &eb_aom_variance64x16_avx2),
-        VarianceParam(64, 32, &eb_aom_variance64x32_c, &eb_aom_variance64x32_avx2),
-        VarianceParam(64, 64, &eb_aom_variance64x64_c, &eb_aom_variance64x64_avx2),
-        VarianceParam(64, 128, &eb_aom_variance64x128_c, &eb_aom_variance64x128_avx2),
-        VarianceParam(128, 64, &eb_aom_variance128x64_c, &eb_aom_variance128x64_avx2),
+        VarianceParam(32, 16, &eb_aom_variance32x16_c,
+                      &eb_aom_variance32x16_avx2),
+        VarianceParam(32, 32, &eb_aom_variance32x32_c,
+                      &eb_aom_variance32x32_avx2),
+        VarianceParam(32, 64, &eb_aom_variance32x64_c,
+                      &eb_aom_variance32x64_avx2),
+        VarianceParam(64, 16, &eb_aom_variance64x16_c,
+                      &eb_aom_variance64x16_avx2),
+        VarianceParam(64, 32, &eb_aom_variance64x32_c,
+                      &eb_aom_variance64x32_avx2),
+        VarianceParam(64, 64, &eb_aom_variance64x64_c,
+                      &eb_aom_variance64x64_avx2),
+        VarianceParam(64, 128, &eb_aom_variance64x128_c,
+                      &eb_aom_variance64x128_avx2),
+        VarianceParam(128, 64, &eb_aom_variance128x64_c,
+                      &eb_aom_variance128x64_avx2),
         VarianceParam(128, 128, &eb_aom_variance128x128_c,
                       &eb_aom_variance128x128_avx2)));
 
-}  // namespace
+typedef std::tuple<int, int> BlockSize;
+BlockSize TEST_BLOCK_SIZES[] = {
+    BlockSize(4, 4),    BlockSize(4, 8),    BlockSize(4, 16),
+    BlockSize(8, 4),    BlockSize(8, 8),    BlockSize(8, 16),
+    BlockSize(8, 32),   BlockSize(16, 4),   BlockSize(16, 8),
+    BlockSize(16, 16),  BlockSize(16, 32),  BlockSize(16, 64),
+    BlockSize(32, 8),   BlockSize(32, 16),  BlockSize(32, 32),
+    BlockSize(32, 64),  BlockSize(64, 16),  BlockSize(64, 32),
+    BlockSize(64, 64),  BlockSize(64, 128), BlockSize(128, 64),
+    BlockSize(128, 128)};
+/**
+ * @brief Unit test for variance functions, target functions include:
+ *  - highbd_variance64_{c,avx2}
+ *
+ * Test strategy:
+ *  This test case contains zero test, random value test, one quarter test as
+ * test pattern.
+ *
+ *
+ * Expect result:
+ *  Results come from reference functon and target function are
+ * equal.
+ *
+ * Test coverage:
+ *
+ * Test cases:
+ *
+ */
+class HBDVarianceTest : public ::testing::TestWithParam<BlockSize> {
```

Comment:
I found the reference code "highbd_variance64_c" is using low bit-depth (8-bit) for variance calculation. The implementation seems incorrect for HBD, attached my test code and results for your reference. 

~~~
// High bit-depth variance test
using HbdVarianceParam = std::tuple<uint32_t,  /**< width */
                                    uint32_t,  /**< height */
                                    uint32_t>; /**< bit-depth */

class HbdVarianceTest : public ::testing::TestWithParam<HbdVarianceParam> {
  public:
    HbdVarianceTest()
        : width_(TEST_GET_PARAM(0)),
          height_(TEST_GET_PARAM(1)),
          bd_(TEST_GET_PARAM(2)) {
        src_data_ = reinterpret_cast<uint16_t *>(
            eb_aom_memalign(32, 2 * MAX_BLOCK_SIZE));
        ref_data_ = reinterpret_cast<uint16_t *>(
            eb_aom_memalign(32, 2 * MAX_BLOCK_SIZE));
    }

    ~HbdVarianceTest() {
        eb_aom_free(src_data_);
        src_data_ = nullptr;
        eb_aom_free(ref_data_);
        ref_data_ = nullptr;
    }

    uint64_t cal_variance(uint64_t sse) {
        int sum = 0;
        for (uint32_t h = 0; h < height_; h++) {
            for (uint32_t w = 0; w < width_; w++) {
                const int diff =
                    src_data_[h * width_ + w] - ref_data_[h * width_ + w];
                sum += diff;
            }
        }
        return ((uint64_t)(sum * sum) / (height_ * width_)) - sse;
    }

    void run_zero_test() {
        for (int i = 0; i < (1 << bd_); ++i) {
            // memset(src_data_, i, MAX_BLOCK_SIZE);
            eb_aom_memset16(src_data_, i, MAX_BLOCK_SIZE);
            for (int j = 0; j < (1 << bd_); ++j) {
                // memset(ref_data_, j, MAX_BLOCK_SIZE);
                eb_aom_memset16(ref_data_, j, MAX_BLOCK_SIZE);
                uint64_t sse64;
                highbd_variance64_avx2(reinterpret_cast<uint8_t *>(src_data_),
                                       width_,
                                       reinterpret_cast<uint8_t *>(ref_data_),
                                       width_,
                                       width_,
                                       height_,
                                       &sse64);
                ASSERT_EQ(cal_variance(sse64), 0UL)
                    << "Variance is mismatched, src values: " << i
                    << " ref values: " << j;
            }
        }
    }

    void run_match_test() {
        SVTRandom rnd(16, false);
        for (int i = 0; i < 10; ++i) {
            for (int j = 0; j < MAX_BLOCK_SIZE; j++) {
                src_data_[j] = rnd.random() && ((1 << bd_) - 1);
                ref_data_[j] = rnd.random() && ((1 << bd_) - 1);
            }
            uint64_t sse64_c, sse64_asm;
            highbd_variance64_c(reinterpret_cast<uint8_t *>(src_data_),
                                width_,
                                reinterpret_cast<uint8_t *>(ref_data_),
                                width_,
                                width_,
                                height_,
                                &sse64_c);
            highbd_variance64_avx2(reinterpret_cast<uint8_t *>(src_data_),
                                   width_,
                                   reinterpret_cast<uint8_t *>(ref_data_),
                                   width_,
                                   width_,
                                   height_,
                                   &sse64_asm);
            ASSERT_EQ(sse64_c, sse64_asm) << "Error at test index: " << i;
        }
    }

    void run_one_quarter_test() {
        const int half = width_ * height_ / 2;
        eb_aom_memset16(src_data_, (1 << bd_) - 1, MAX_BLOCK_SIZE);
        eb_aom_memset16(ref_data_, (1 << bd_) - 1, half);
        eb_aom_memset16(ref_data_ + half, 0, half);
        uint64_t sse64_asm, expected;
        highbd_variance64_avx2(reinterpret_cast<uint8_t *>(src_data_),
                               width_,
                               reinterpret_cast<uint8_t *>(ref_data_),
                               width_,
                               width_,
                               height_,
                               &sse64_asm);
        expected = width_ * height_ * ((1 << bd_) - 1) * ((1 << bd_) - 1) / 4;
        ASSERT_EQ(cal_variance(sse64_asm), expected);
    }

  private:
    uint16_t *src_data_;
    uint16_t *ref_data_;
    uint32_t width_;
    uint32_t height_;
    uint32_t bd_;
};

TEST_P(HbdVarianceTest, ZeroTest) {
    run_zero_test();
};

TEST_P(HbdVarianceTest, MatchTest) {
    run_match_test();
};

TEST_P(HbdVarianceTest, OneQuarterTest) {
    run_one_quarter_test();
};

static const uint32_t WIDTH_TABLE[] = {4, 8, 16, 32, 64, 128};
static const uint32_t HEIGHT_TABLE[] = {4, 8, 16, 32, 64, 128};
static const uint32_t BD_TABLE[] = {8, 10, 12, 16};

INSTANTIATE_TEST_CASE_P(Variance, HbdVarianceTest,
                        ::testing::Combine(::testing::ValuesIn(WIDTH_TABLE),
                                           ::testing::ValuesIn(HEIGHT_TABLE),
                                           ::testing::ValuesIn(BD_TABLE)));
~~~
~~~
Running main() from C:\Works\AV1\SVT-AV1\third_party\googletest\src\gtest_main.cc
Note: Google Test filter = */HbdVarianceTest.*
[==========] Running 432 tests from 1 test case.
[----------] Global test environment set-up.
[----------] 432 tests from Variance/HbdVarianceTest
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/0
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 8
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/0, where GetParam() = (4, 4, 8) (2 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/1
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 8
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/1, where GetParam() = (4, 4, 10) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/2
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 8
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/2, where GetParam() = (4, 4, 12) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/3
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 8
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/3, where GetParam() = (4, 4, 16) (3 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/4
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 16
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/4, where GetParam() = (4, 8, 8) (2 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/5
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 16
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/5, where GetParam() = (4, 8, 10) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/6
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 16
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/6, where GetParam() = (4, 8, 12) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/7
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 16
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/7, where GetParam() = (4, 8, 16) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/8
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 32
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/8, where GetParam() = (4, 16, 8) (2 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/9
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 32
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/9, where GetParam() = (4, 16, 10) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/10
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 32
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/10, where GetParam() = (4, 16, 12) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/11
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 32
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/11, where GetParam() = (4, 16, 16) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/12
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 64
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/12, where GetParam() = (4, 32, 8) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/13
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 64
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/13, where GetParam() = (4, 32, 10) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/14
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 64
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/14, where GetParam() = (4, 32, 12) (2 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/15
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 64
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/15, where GetParam() = (4, 32, 16) (2 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/16
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 128
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/16, where GetParam() = (4, 64, 8) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/17
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 128
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/17, where GetParam() = (4, 64, 10) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/18
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 128
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/18, where GetParam() = (4, 64, 12) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/19
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 128
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/19, where GetParam() = (4, 64, 16) (2 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/20
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 256
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/20, where GetParam() = (4, 128, 8) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/21
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 256
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/21, where GetParam() = (4, 128, 10) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/22
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 256
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/22, where GetParam() = (4, 128, 12) (2 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/23
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 256
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/23, where GetParam() = (4, 128, 16) (2 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/24
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 16
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/24, where GetParam() = (8, 4, 8) (3 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/25
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 16
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/25, where GetParam() = (8, 4, 10) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/26
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 16
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/26, where GetParam() = (8, 4, 12) (2 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/27
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 16
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/27, where GetParam() = (8, 4, 16) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/28
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 32
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/28, where GetParam() = (8, 8, 8) (4 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/29
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 32
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/29, where GetParam() = (8, 8, 10) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/30
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 32
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/30, where GetParam() = (8, 8, 12) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/31
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 32
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/31, where GetParam() = (8, 8, 16) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/32
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 64
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/32, where GetParam() = (8, 16, 8) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/33
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 64
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/33, where GetParam() = (8, 16, 10) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/34
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 64
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/34, where GetParam() = (8, 16, 12) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/35
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 64
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/35, where GetParam() = (8, 16, 16) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/36
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 128
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/36, where GetParam() = (8, 32, 8) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/37
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 128
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/37, where GetParam() = (8, 32, 10) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/38
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 128
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/38, where GetParam() = (8, 32, 12) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/39
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 128
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/39, where GetParam() = (8, 32, 16) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/40
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 256
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/40, where GetParam() = (8, 64, 8) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/41
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 256
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/41, where GetParam() = (8, 64, 10) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/42
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 256
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/42, where GetParam() = (8, 64, 12) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/43
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 256
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/43, where GetParam() = (8, 64, 16) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/44
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 512
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/44, where GetParam() = (8, 128, 8) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/45
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 512
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/45, where GetParam() = (8, 128, 10) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/46
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 512
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/46, where GetParam() = (8, 128, 12) (2 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/47
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 512
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/47, where GetParam() = (8, 128, 16) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/48
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 32
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/48, where GetParam() = (16, 4, 8) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/49
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 32
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/49, where GetParam() = (16, 4, 10) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/50
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 32
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/50, where GetParam() = (16, 4, 12) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/51
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 32
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/51, where GetParam() = (16, 4, 16) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/52
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 64
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/52, where GetParam() = (16, 8, 8) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/53
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 64
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/53, where GetParam() = (16, 8, 10) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/54
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 64
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/54, where GetParam() = (16, 8, 12) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/55
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 64
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/55, where GetParam() = (16, 8, 16) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/56
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 128
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/56, where GetParam() = (16, 16, 8) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/57
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 128
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/57, where GetParam() = (16, 16, 10) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/58
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 128
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/58, where GetParam() = (16, 16, 12) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/59
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 128
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/59, where GetParam() = (16, 16, 16) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/60
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 256
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/60, where GetParam() = (16, 32, 8) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/61
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 256
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/61, where GetParam() = (16, 32, 10) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/62
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 256
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/62, where GetParam() = (16, 32, 12) (2 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/63
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 256
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/63, where GetParam() = (16, 32, 16) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/64
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 512
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/64, where GetParam() = (16, 64, 8) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/65
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 512
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/65, where GetParam() = (16, 64, 10) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/66
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 512
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/66, where GetParam() = (16, 64, 12) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/67
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 512
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/67, where GetParam() = (16, 64, 16) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/68
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 1024
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/68, where GetParam() = (16, 128, 8) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/69
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 1024
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/69, where GetParam() = (16, 128, 10) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/70
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 1024
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/70, where GetParam() = (16, 128, 12) (2 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/71
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 1024
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/71, where GetParam() = (16, 128, 16) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/72
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 64
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/72, where GetParam() = (32, 4, 8) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/73
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 64
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/73, where GetParam() = (32, 4, 10) (2 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/74
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 64
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/74, where GetParam() = (32, 4, 12) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/75
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 64
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/75, where GetParam() = (32, 4, 16) (2 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/76
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 128
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/76, where GetParam() = (32, 8, 8) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/77
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 128
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/77, where GetParam() = (32, 8, 10) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/78
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 128
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/78, where GetParam() = (32, 8, 12) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/79
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 128
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/79, where GetParam() = (32, 8, 16) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/80
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 256
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/80, where GetParam() = (32, 16, 8) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/81
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 256
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/81, where GetParam() = (32, 16, 10) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/82
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 256
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/82, where GetParam() = (32, 16, 12) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/83
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 256
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/83, where GetParam() = (32, 16, 16) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/84
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 512
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/84, where GetParam() = (32, 32, 8) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/85
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 512
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/85, where GetParam() = (32, 32, 10) (2 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/86
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 512
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/86, where GetParam() = (32, 32, 12) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/87
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 512
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/87, where GetParam() = (32, 32, 16) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/88
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 1024
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/88, where GetParam() = (32, 64, 8) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/89
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 1024
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/89, where GetParam() = (32, 64, 10) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/90
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 1024
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/90, where GetParam() = (32, 64, 12) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/91
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 1024
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/91, where GetParam() = (32, 64, 16) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/92
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 2048
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/92, where GetParam() = (32, 128, 8) (2 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/93
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 2048
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/93, where GetParam() = (32, 128, 10) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/94
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 2048
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/94, where GetParam() = (32, 128, 12) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/95
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 2048
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/95, where GetParam() = (32, 128, 16) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/96
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 128
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/96, where GetParam() = (64, 4, 8) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/97
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 128
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/97, where GetParam() = (64, 4, 10) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/98
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 128
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/98, where GetParam() = (64, 4, 12) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/99
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 128
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/99, where GetParam() = (64, 4, 16) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/100
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 256
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/100, where GetParam() = (64, 8, 8) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/101
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 256
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/101, where GetParam() = (64, 8, 10) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/102
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 256
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/102, where GetParam() = (64, 8, 12) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/103
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 256
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/103, where GetParam() = (64, 8, 16) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/104
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 512
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/104, where GetParam() = (64, 16, 8) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/105
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 512
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/105, where GetParam() = (64, 16, 10) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/106
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 512
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/106, where GetParam() = (64, 16, 12) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/107
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 512
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/107, where GetParam() = (64, 16, 16) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/108
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 1024
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/108, where GetParam() = (64, 32, 8) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/109
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 1024
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/109, where GetParam() = (64, 32, 10) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/110
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 1024
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/110, where GetParam() = (64, 32, 12) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/111
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 1024
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/111, where GetParam() = (64, 32, 16) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/112
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 2048
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/112, where GetParam() = (64, 64, 8) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/113
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 2048
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/113, where GetParam() = (64, 64, 10) (2 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/114
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 2048
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/114, where GetParam() = (64, 64, 12) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/115
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 2048
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/115, where GetParam() = (64, 64, 16) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/116
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 4096
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/116, where GetParam() = (64, 128, 8) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/117
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 4096
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/117, where GetParam() = (64, 128, 10) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/118
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 4096
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/118, where GetParam() = (64, 128, 12) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/119
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 4096
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/119, where GetParam() = (64, 128, 16) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/120
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 18446744073709551104
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/120, where GetParam() = (128, 4, 8) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/121
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 18446744073709551104
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/121, where GetParam() = (128, 4, 10) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/122
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 18446744073709551104
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/122, where GetParam() = (128, 4, 12) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/123
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 18446744073709551104
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/123, where GetParam() = (128, 4, 16) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/124
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 18014398505287680
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 46
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/124, where GetParam() = (128, 8, 8) (3 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/125
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 18014398505287680
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 46
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/125, where GetParam() = (128, 8, 10) (3 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/126
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 18014398505287680
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 46
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/126, where GetParam() = (128, 8, 12) (3 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/127
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 18014398505287680
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 46
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/127, where GetParam() = (128, 8, 16) (3 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/128
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 1024
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/128, where GetParam() = (128, 16, 8) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/129
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 1024
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/129, where GetParam() = (128, 16, 10) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/130
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 1024
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/130, where GetParam() = (128, 16, 12) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/131
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 1024
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/131, where GetParam() = (128, 16, 16) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/132
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 2048
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/132, where GetParam() = (128, 32, 8) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/133
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 2048
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/133, where GetParam() = (128, 32, 10) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/134
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 2048
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/134, where GetParam() = (128, 32, 12) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/135
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 2048
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/135, where GetParam() = (128, 32, 16) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/136
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 4096
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/136, where GetParam() = (128, 64, 8) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/137
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 4096
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/137, where GetParam() = (128, 64, 10) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/138
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 4096
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/138, where GetParam() = (128, 64, 12) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/139
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 4096
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/139, where GetParam() = (128, 64, 16) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/140
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 8192
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/140, where GetParam() = (128, 128, 8) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/141
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 8192
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/141, where GetParam() = (128, 128, 10) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/142
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 8192
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/142, where GetParam() = (128, 128, 12) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.ZeroTest/143
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(421): error: Expected equality of these values:
  cal_variance(sse64)
    Which is: 8192
  0UL
    Which is: 0
Variance is mismatched, src values: 0 ref values: 1
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/143, where GetParam() = (128, 128, 16) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/0
[       OK ] Variance/HbdVarianceTest.MatchTest/0 (48 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/1
[       OK ] Variance/HbdVarianceTest.MatchTest/1 (49 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/2
[       OK ] Variance/HbdVarianceTest.MatchTest/2 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/3
[       OK ] Variance/HbdVarianceTest.MatchTest/3 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/4
[       OK ] Variance/HbdVarianceTest.MatchTest/4 (48 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/5
[       OK ] Variance/HbdVarianceTest.MatchTest/5 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/6
[       OK ] Variance/HbdVarianceTest.MatchTest/6 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/7
[       OK ] Variance/HbdVarianceTest.MatchTest/7 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/8
[       OK ] Variance/HbdVarianceTest.MatchTest/8 (50 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/9
[       OK ] Variance/HbdVarianceTest.MatchTest/9 (48 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/10
[       OK ] Variance/HbdVarianceTest.MatchTest/10 (48 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/11
[       OK ] Variance/HbdVarianceTest.MatchTest/11 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/12
[       OK ] Variance/HbdVarianceTest.MatchTest/12 (49 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/13
[       OK ] Variance/HbdVarianceTest.MatchTest/13 (48 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/14
[       OK ] Variance/HbdVarianceTest.MatchTest/14 (48 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/15
[       OK ] Variance/HbdVarianceTest.MatchTest/15 (46 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/16
[       OK ] Variance/HbdVarianceTest.MatchTest/16 (46 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/17
[       OK ] Variance/HbdVarianceTest.MatchTest/17 (49 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/18
[       OK ] Variance/HbdVarianceTest.MatchTest/18 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/19
[       OK ] Variance/HbdVarianceTest.MatchTest/19 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/20
[       OK ] Variance/HbdVarianceTest.MatchTest/20 (46 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/21
[       OK ] Variance/HbdVarianceTest.MatchTest/21 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/22
[       OK ] Variance/HbdVarianceTest.MatchTest/22 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/23
[       OK ] Variance/HbdVarianceTest.MatchTest/23 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/24
[       OK ] Variance/HbdVarianceTest.MatchTest/24 (48 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/25
[       OK ] Variance/HbdVarianceTest.MatchTest/25 (49 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/26
[       OK ] Variance/HbdVarianceTest.MatchTest/26 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/27
[       OK ] Variance/HbdVarianceTest.MatchTest/27 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/28
[       OK ] Variance/HbdVarianceTest.MatchTest/28 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/29
[       OK ] Variance/HbdVarianceTest.MatchTest/29 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/30
[       OK ] Variance/HbdVarianceTest.MatchTest/30 (48 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/31
[       OK ] Variance/HbdVarianceTest.MatchTest/31 (46 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/32
[       OK ] Variance/HbdVarianceTest.MatchTest/32 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/33
[       OK ] Variance/HbdVarianceTest.MatchTest/33 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/34
[       OK ] Variance/HbdVarianceTest.MatchTest/34 (54 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/35
[       OK ] Variance/HbdVarianceTest.MatchTest/35 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/36
[       OK ] Variance/HbdVarianceTest.MatchTest/36 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/37
[       OK ] Variance/HbdVarianceTest.MatchTest/37 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/38
[       OK ] Variance/HbdVarianceTest.MatchTest/38 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/39
[       OK ] Variance/HbdVarianceTest.MatchTest/39 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/40
[       OK ] Variance/HbdVarianceTest.MatchTest/40 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/41
[       OK ] Variance/HbdVarianceTest.MatchTest/41 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/42
[       OK ] Variance/HbdVarianceTest.MatchTest/42 (46 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/43
[       OK ] Variance/HbdVarianceTest.MatchTest/43 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/44
[       OK ] Variance/HbdVarianceTest.MatchTest/44 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/45
[       OK ] Variance/HbdVarianceTest.MatchTest/45 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/46
[       OK ] Variance/HbdVarianceTest.MatchTest/46 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/47
[       OK ] Variance/HbdVarianceTest.MatchTest/47 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/48
[       OK ] Variance/HbdVarianceTest.MatchTest/48 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/49
[       OK ] Variance/HbdVarianceTest.MatchTest/49 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/50
[       OK ] Variance/HbdVarianceTest.MatchTest/50 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/51
[       OK ] Variance/HbdVarianceTest.MatchTest/51 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/52
[       OK ] Variance/HbdVarianceTest.MatchTest/52 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/53
[       OK ] Variance/HbdVarianceTest.MatchTest/53 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/54
[       OK ] Variance/HbdVarianceTest.MatchTest/54 (49 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/55
[       OK ] Variance/HbdVarianceTest.MatchTest/55 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/56
[       OK ] Variance/HbdVarianceTest.MatchTest/56 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/57
[       OK ] Variance/HbdVarianceTest.MatchTest/57 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/58
[       OK ] Variance/HbdVarianceTest.MatchTest/58 (46 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/59
[       OK ] Variance/HbdVarianceTest.MatchTest/59 (48 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/60
[       OK ] Variance/HbdVarianceTest.MatchTest/60 (48 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/61
[       OK ] Variance/HbdVarianceTest.MatchTest/61 (46 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/62
[       OK ] Variance/HbdVarianceTest.MatchTest/62 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/63
[       OK ] Variance/HbdVarianceTest.MatchTest/63 (48 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/64
[       OK ] Variance/HbdVarianceTest.MatchTest/64 (46 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/65
[       OK ] Variance/HbdVarianceTest.MatchTest/65 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/66
[       OK ] Variance/HbdVarianceTest.MatchTest/66 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/67
[       OK ] Variance/HbdVarianceTest.MatchTest/67 (46 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/68
[       OK ] Variance/HbdVarianceTest.MatchTest/68 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/69
[       OK ] Variance/HbdVarianceTest.MatchTest/69 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/70
[       OK ] Variance/HbdVarianceTest.MatchTest/70 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/71
[       OK ] Variance/HbdVarianceTest.MatchTest/71 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/72
[       OK ] Variance/HbdVarianceTest.MatchTest/72 (46 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/73
[       OK ] Variance/HbdVarianceTest.MatchTest/73 (46 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/74
[       OK ] Variance/HbdVarianceTest.MatchTest/74 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/75
[       OK ] Variance/HbdVarianceTest.MatchTest/75 (48 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/76
[       OK ] Variance/HbdVarianceTest.MatchTest/76 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/77
[       OK ] Variance/HbdVarianceTest.MatchTest/77 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/78
[       OK ] Variance/HbdVarianceTest.MatchTest/78 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/79
[       OK ] Variance/HbdVarianceTest.MatchTest/79 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/80
[       OK ] Variance/HbdVarianceTest.MatchTest/80 (48 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/81
[       OK ] Variance/HbdVarianceTest.MatchTest/81 (48 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/82
[       OK ] Variance/HbdVarianceTest.MatchTest/82 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/83
[       OK ] Variance/HbdVarianceTest.MatchTest/83 (48 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/84
[       OK ] Variance/HbdVarianceTest.MatchTest/84 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/85
[       OK ] Variance/HbdVarianceTest.MatchTest/85 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/86
[       OK ] Variance/HbdVarianceTest.MatchTest/86 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/87
[       OK ] Variance/HbdVarianceTest.MatchTest/87 (46 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/88
[       OK ] Variance/HbdVarianceTest.MatchTest/88 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/89
[       OK ] Variance/HbdVarianceTest.MatchTest/89 (46 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/90
[       OK ] Variance/HbdVarianceTest.MatchTest/90 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/91
[       OK ] Variance/HbdVarianceTest.MatchTest/91 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/92
[       OK ] Variance/HbdVarianceTest.MatchTest/92 (48 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/93
[       OK ] Variance/HbdVarianceTest.MatchTest/93 (48 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/94
[       OK ] Variance/HbdVarianceTest.MatchTest/94 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/95
[       OK ] Variance/HbdVarianceTest.MatchTest/95 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/96
[       OK ] Variance/HbdVarianceTest.MatchTest/96 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/97
[       OK ] Variance/HbdVarianceTest.MatchTest/97 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/98
[       OK ] Variance/HbdVarianceTest.MatchTest/98 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/99
[       OK ] Variance/HbdVarianceTest.MatchTest/99 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/100
[       OK ] Variance/HbdVarianceTest.MatchTest/100 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/101
[       OK ] Variance/HbdVarianceTest.MatchTest/101 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/102
[       OK ] Variance/HbdVarianceTest.MatchTest/102 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/103
[       OK ] Variance/HbdVarianceTest.MatchTest/103 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/104
[       OK ] Variance/HbdVarianceTest.MatchTest/104 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/105
[       OK ] Variance/HbdVarianceTest.MatchTest/105 (48 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/106
[       OK ] Variance/HbdVarianceTest.MatchTest/106 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/107
[       OK ] Variance/HbdVarianceTest.MatchTest/107 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/108
[       OK ] Variance/HbdVarianceTest.MatchTest/108 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/109
[       OK ] Variance/HbdVarianceTest.MatchTest/109 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/110
[       OK ] Variance/HbdVarianceTest.MatchTest/110 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/111
[       OK ] Variance/HbdVarianceTest.MatchTest/111 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/112
[       OK ] Variance/HbdVarianceTest.MatchTest/112 (46 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/113
[       OK ] Variance/HbdVarianceTest.MatchTest/113 (48 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/114
[       OK ] Variance/HbdVarianceTest.MatchTest/114 (48 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/115
[       OK ] Variance/HbdVarianceTest.MatchTest/115 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/116
[       OK ] Variance/HbdVarianceTest.MatchTest/116 (46 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/117
[       OK ] Variance/HbdVarianceTest.MatchTest/117 (48 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/118
[       OK ] Variance/HbdVarianceTest.MatchTest/118 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/119
[       OK ] Variance/HbdVarianceTest.MatchTest/119 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/120
[       OK ] Variance/HbdVarianceTest.MatchTest/120 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/121
[       OK ] Variance/HbdVarianceTest.MatchTest/121 (50 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/122
[       OK ] Variance/HbdVarianceTest.MatchTest/122 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/123
[       OK ] Variance/HbdVarianceTest.MatchTest/123 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/124
[       OK ] Variance/HbdVarianceTest.MatchTest/124 (46 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/125
[       OK ] Variance/HbdVarianceTest.MatchTest/125 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/126
[       OK ] Variance/HbdVarianceTest.MatchTest/126 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/127
[       OK ] Variance/HbdVarianceTest.MatchTest/127 (46 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/128
[       OK ] Variance/HbdVarianceTest.MatchTest/128 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/129
[       OK ] Variance/HbdVarianceTest.MatchTest/129 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/130
[       OK ] Variance/HbdVarianceTest.MatchTest/130 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/131
[       OK ] Variance/HbdVarianceTest.MatchTest/131 (46 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/132
[       OK ] Variance/HbdVarianceTest.MatchTest/132 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/133
[       OK ] Variance/HbdVarianceTest.MatchTest/133 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/134
[       OK ] Variance/HbdVarianceTest.MatchTest/134 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/135
[       OK ] Variance/HbdVarianceTest.MatchTest/135 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/136
[       OK ] Variance/HbdVarianceTest.MatchTest/136 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/137
[       OK ] Variance/HbdVarianceTest.MatchTest/137 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/138
[       OK ] Variance/HbdVarianceTest.MatchTest/138 (49 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/139
[       OK ] Variance/HbdVarianceTest.MatchTest/139 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/140
[       OK ] Variance/HbdVarianceTest.MatchTest/140 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/141
[       OK ] Variance/HbdVarianceTest.MatchTest/141 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/142
[       OK ] Variance/HbdVarianceTest.MatchTest/142 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.MatchTest/143
[       OK ] Variance/HbdVarianceTest.MatchTest/143 (47 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/0
[       OK ] Variance/HbdVarianceTest.OneQuarterTest/0 (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/1
[       OK ] Variance/HbdVarianceTest.OneQuarterTest/1 (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/2
[       OK ] Variance/HbdVarianceTest.OneQuarterTest/2 (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/3
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 1152921504606322692
  expected
    Which is: 1073217540
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/3, where GetParam() = (4, 4, 16) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/4
[       OK ] Variance/HbdVarianceTest.OneQuarterTest/4 (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/5
[       OK ] Variance/HbdVarianceTest.OneQuarterTest/5 (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/6
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 576460752303357960
  expected
    Which is: 134152200
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/6, where GetParam() = (4, 8, 12) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/7
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 576460752302374920
  expected
    Which is: 1072693256
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/7, where GetParam() = (4, 8, 16) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/8
[       OK ] Variance/HbdVarianceTest.OneQuarterTest/8 (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/9
[       OK ] Variance/HbdVarianceTest.OneQuarterTest/9 (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/10
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 288230376151580688
  expected
    Which is: 268304400
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/10, where GetParam() = (4, 16, 12) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/11
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 288230376149614608
  expected
    Which is: 1071644688
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/11, where GetParam() = (4, 16, 16) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/12
[       OK ] Variance/HbdVarianceTest.OneQuarterTest/12 (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/13
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 144115188075790368
  expected
    Which is: 33488928
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/13, where GetParam() = (4, 32, 10) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/14
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 144115188075593760
  expected
    Which is: 536608800
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/14, where GetParam() = (4, 32, 12) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/15
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 144115188071661600
  expected
    Which is: 1069547552
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/15, where GetParam() = (4, 32, 16) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/16
[       OK ] Variance/HbdVarianceTest.OneQuarterTest/16 (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/17
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 72057594037796928
  expected
    Which is: 66977856
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/17, where GetParam() = (4, 64, 10) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/18
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 72057594037403712
  expected
    Which is: 1073217600
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/18, where GetParam() = (4, 64, 12) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/19
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 72057594029539392
  expected
    Which is: 1065353280
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/19, where GetParam() = (4, 64, 16) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/20
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 36028797018898560
  expected
    Which is: 8323200
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/20, where GetParam() = (4, 128, 8) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/21
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 36028797018701952
  expected
    Which is: 133955712
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/21, where GetParam() = (4, 128, 10) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/22
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 36028797017915520
  expected
    Which is: 1072693376
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/22, where GetParam() = (4, 128, 12) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/23
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 128
  expected
    Which is: 1056964736
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/23, where GetParam() = (4, 128, 16) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/24
[       OK ] Variance/HbdVarianceTest.OneQuarterTest/24 (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/25
[       OK ] Variance/HbdVarianceTest.OneQuarterTest/25 (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/26
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 576460752303357960
  expected
    Which is: 134152200
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/26, where GetParam() = (8, 4, 12) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/27
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 576460752302374920
  expected
    Which is: 1072693256
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/27, where GetParam() = (8, 4, 16) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/28
[       OK ] Variance/HbdVarianceTest.OneQuarterTest/28 (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/29
[       OK ] Variance/HbdVarianceTest.OneQuarterTest/29 (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/30
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 288230376151580688
  expected
    Which is: 268304400
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/30, where GetParam() = (8, 8, 12) (3 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/31
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 288230376149614608
  expected
    Which is: 1071644688
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/31, where GetParam() = (8, 8, 16) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/32
[       OK ] Variance/HbdVarianceTest.OneQuarterTest/32 (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/33
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 144115188075790368
  expected
    Which is: 33488928
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/33, where GetParam() = (8, 16, 10) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/34
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 144115188075593760
  expected
    Which is: 536608800
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/34, where GetParam() = (8, 16, 12) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/35
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 144115188071661600
  expected
    Which is: 1069547552
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/35, where GetParam() = (8, 16, 16) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/36
[       OK ] Variance/HbdVarianceTest.OneQuarterTest/36 (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/37
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 72057594037796928
  expected
    Which is: 66977856
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/37, where GetParam() = (8, 32, 10) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/38
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 72057594037403712
  expected
    Which is: 1073217600
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/38, where GetParam() = (8, 32, 12) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/39
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 72057594029539392
  expected
    Which is: 1065353280
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/39, where GetParam() = (8, 32, 16) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/40
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 36028797018898560
  expected
    Which is: 8323200
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/40, where GetParam() = (8, 64, 8) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/41
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 36028797018701952
  expected
    Which is: 133955712
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/41, where GetParam() = (8, 64, 10) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/42
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 36028797017915520
  expected
    Which is: 1072693376
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/42, where GetParam() = (8, 64, 12) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/43
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 128
  expected
    Which is: 1056964736
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/43, where GetParam() = (8, 64, 16) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/44
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 18014398509351168
  expected
    Which is: 16646400
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/44, where GetParam() = (8, 128, 8) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/45
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 18014398508957952
  expected
    Which is: 267911424
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/45, where GetParam() = (8, 128, 10) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/46
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 18014398507385088
  expected
    Which is: 1071644928
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/46, where GetParam() = (8, 128, 12) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/47
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 256
  expected
    Which is: 1040187648
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/47, where GetParam() = (8, 128, 16) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/48
[       OK ] Variance/HbdVarianceTest.OneQuarterTest/48 (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/49
[       OK ] Variance/HbdVarianceTest.OneQuarterTest/49 (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/50
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 288230376151580688
  expected
    Which is: 268304400
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/50, where GetParam() = (16, 4, 12) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/51
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 288230376149614608
  expected
    Which is: 1071644688
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/51, where GetParam() = (16, 4, 16) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/52
[       OK ] Variance/HbdVarianceTest.OneQuarterTest/52 (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/53
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 144115188075790368
  expected
    Which is: 33488928
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/53, where GetParam() = (16, 8, 10) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/54
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 144115188075593760
  expected
    Which is: 536608800
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/54, where GetParam() = (16, 8, 12) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/55
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 144115188071661600
  expected
    Which is: 1069547552
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/55, where GetParam() = (16, 8, 16) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/56
[       OK ] Variance/HbdVarianceTest.OneQuarterTest/56 (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/57
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 72057594037796928
  expected
    Which is: 66977856
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/57, where GetParam() = (16, 16, 10) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/58
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 72057594037403712
  expected
    Which is: 1073217600
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/58, where GetParam() = (16, 16, 12) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/59
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 72057594029539392
  expected
    Which is: 1065353280
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/59, where GetParam() = (16, 16, 16) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/60
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 36028797018898560
  expected
    Which is: 8323200
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/60, where GetParam() = (16, 32, 8) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/61
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 36028797018701952
  expected
    Which is: 133955712
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/61, where GetParam() = (16, 32, 10) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/62
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 36028797017915520
  expected
    Which is: 1072693376
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/62, where GetParam() = (16, 32, 12) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/63
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 128
  expected
    Which is: 1056964736
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/63, where GetParam() = (16, 32, 16) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/64
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 18014398509351168
  expected
    Which is: 16646400
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/64, where GetParam() = (16, 64, 8) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/65
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 18014398508957952
  expected
    Which is: 267911424
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/65, where GetParam() = (16, 64, 10) (2 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/66
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 18014398507385088
  expected
    Which is: 1071644928
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/66, where GetParam() = (16, 64, 12) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/67
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 256
  expected
    Which is: 1040187648
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/67, where GetParam() = (16, 64, 16) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/68
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 9007199254479360
  expected
    Which is: 33292800
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/68, where GetParam() = (16, 128, 8) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/69
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 9007199253692928
  expected
    Which is: 535822848
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/69, where GetParam() = (16, 128, 10) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/70
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 512
  expected
    Which is: 1069548032
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/70, where GetParam() = (16, 128, 12) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/71
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 512
  expected
    Which is: 1006633472
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/71, where GetParam() = (16, 128, 16) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/72
[       OK ] Variance/HbdVarianceTest.OneQuarterTest/72 (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/73
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 144115188075790368
  expected
    Which is: 33488928
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/73, where GetParam() = (32, 4, 10) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/74
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 144115188075593760
  expected
    Which is: 536608800
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/74, where GetParam() = (32, 4, 12) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/75
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 144115188071661600
  expected
    Which is: 1069547552
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/75, where GetParam() = (32, 4, 16) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/76
[       OK ] Variance/HbdVarianceTest.OneQuarterTest/76 (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/77
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 72057594037796928
  expected
    Which is: 66977856
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/77, where GetParam() = (32, 8, 10) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/78
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 72057594037403712
  expected
    Which is: 1073217600
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/78, where GetParam() = (32, 8, 12) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/79
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 72057594029539392
  expected
    Which is: 1065353280
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/79, where GetParam() = (32, 8, 16) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/80
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 36028797018898560
  expected
    Which is: 8323200
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/80, where GetParam() = (32, 16, 8) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/81
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 36028797018701952
  expected
    Which is: 133955712
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/81, where GetParam() = (32, 16, 10) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/82
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 36028797017915520
  expected
    Which is: 1072693376
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/82, where GetParam() = (32, 16, 12) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/83
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 128
  expected
    Which is: 1056964736
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/83, where GetParam() = (32, 16, 16) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/84
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 18014398509351168
  expected
    Which is: 16646400
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/84, where GetParam() = (32, 32, 8) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/85
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 18014398508957952
  expected
    Which is: 267911424
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/85, where GetParam() = (32, 32, 10) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/86
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 18014398507385088
  expected
    Which is: 1071644928
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/86, where GetParam() = (32, 32, 12) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/87
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 256
  expected
    Which is: 1040187648
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/87, where GetParam() = (32, 32, 16) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/88
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 9007199254479360
  expected
    Which is: 33292800
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/88, where GetParam() = (32, 64, 8) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/89
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 9007199253692928
  expected
    Which is: 535822848
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/89, where GetParam() = (32, 64, 10) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/90
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 512
  expected
    Which is: 1069548032
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/90, where GetParam() = (32, 64, 12) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/91
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 512
  expected
    Which is: 1006633472
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/91, where GetParam() = (32, 64, 16) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/92
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 4503599626847232
  expected
    Which is: 66585600
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/92, where GetParam() = (32, 128, 8) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/93
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 1024
  expected
    Which is: 1071645696
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/93, where GetParam() = (32, 128, 10) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/94
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 1024
  expected
    Which is: 1065354240
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/94, where GetParam() = (32, 128, 12) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/95
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 1024
  expected
    Which is: 939525120
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/95, where GetParam() = (32, 128, 16) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/96
[       OK ] Variance/HbdVarianceTest.OneQuarterTest/96 (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/97
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 72057594037796928
  expected
    Which is: 66977856
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/97, where GetParam() = (64, 4, 10) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/98
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 72057594037403712
  expected
    Which is: 1073217600
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/98, where GetParam() = (64, 4, 12) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/99
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 72057594029539392
  expected
    Which is: 1065353280
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/99, where GetParam() = (64, 4, 16) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/100
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 36028797018898560
  expected
    Which is: 8323200
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/100, where GetParam() = (64, 8, 8) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/101
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 36028797018701952
  expected
    Which is: 133955712
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/101, where GetParam() = (64, 8, 10) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/102
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 36028797017915520
  expected
    Which is: 1072693376
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/102, where GetParam() = (64, 8, 12) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/103
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 128
  expected
    Which is: 1056964736
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/103, where GetParam() = (64, 8, 16) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/104
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 18014398509351168
  expected
    Which is: 16646400
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/104, where GetParam() = (64, 16, 8) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/105
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 18014398508957952
  expected
    Which is: 267911424
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/105, where GetParam() = (64, 16, 10) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/106
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 18014398507385088
  expected
    Which is: 1071644928
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/106, where GetParam() = (64, 16, 12) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/107
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 256
  expected
    Which is: 1040187648
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/107, where GetParam() = (64, 16, 16) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/108
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 9007199254479360
  expected
    Which is: 33292800
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/108, where GetParam() = (64, 32, 8) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/109
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 9007199253692928
  expected
    Which is: 535822848
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/109, where GetParam() = (64, 32, 10) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/110
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 512
  expected
    Which is: 1069548032
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/110, where GetParam() = (64, 32, 12) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/111
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 512
  expected
    Which is: 1006633472
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/111, where GetParam() = (64, 32, 16) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/112
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 4503599626847232
  expected
    Which is: 66585600
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/112, where GetParam() = (64, 64, 8) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/113
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 1024
  expected
    Which is: 1071645696
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/113, where GetParam() = (64, 64, 10) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/114
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 1024
  expected
    Which is: 1065354240
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/114, where GetParam() = (64, 64, 12) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/115
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 1024
  expected
    Which is: 939525120
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/115, where GetParam() = (64, 64, 16) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/116
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 2048
  expected
    Which is: 133171200
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/116, where GetParam() = (64, 128, 8) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/117
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 2048
  expected
    Which is: 1069549568
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/117, where GetParam() = (64, 128, 10) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/118
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 2048
  expected
    Which is: 1056966656
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/118, where GetParam() = (64, 128, 12) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/119
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 2048
  expected
    Which is: 805308416
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/119, where GetParam() = (64, 128, 16) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/120
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 36028796979455360
  expected
    Which is: 8323200
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/120, where GetParam() = (128, 4, 8) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/121
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 36028796979881600
  expected
    Which is: 133955712
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/121, where GetParam() = (128, 4, 10) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/122
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 36028796981448320
  expected
    Which is: 1072693376
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/122, where GetParam() = (128, 4, 12) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/123
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 18446744073673698944
  expected
    Which is: 1056964736
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/123, where GetParam() = (128, 4, 16) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/124
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 18014398476058368
  expected
    Which is: 16646400
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/124, where GetParam() = (128, 8, 8) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/125
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 18014398475660544
  expected
    Which is: 267911424
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/125, where GetParam() = (128, 8, 10) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/126
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 18014398473977088
  expected
    Which is: 1071644928
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/126, where GetParam() = (128, 8, 12) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/127
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 18446744073642966272
  expected
    Which is: 1040187648
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/127, where GetParam() = (128, 8, 16) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/128
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 9007199254479360
  expected
    Which is: 33292800
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/128, where GetParam() = (128, 16, 8) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/129
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 9007199253692928
  expected
    Which is: 535822848
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/129, where GetParam() = (128, 16, 10) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/130
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 512
  expected
    Which is: 1069548032
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/130, where GetParam() = (128, 16, 12) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/131
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 512
  expected
    Which is: 1006633472
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/131, where GetParam() = (128, 16, 16) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/132
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 4503599626847232
  expected
    Which is: 66585600
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/132, where GetParam() = (128, 32, 8) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/133
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 1024
  expected
    Which is: 1071645696
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/133, where GetParam() = (128, 32, 10) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/134
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 1024
  expected
    Which is: 1065354240
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/134, where GetParam() = (128, 32, 12) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/135
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 1024
  expected
    Which is: 939525120
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/135, where GetParam() = (128, 32, 16) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/136
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 2048
  expected
    Which is: 133171200
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/136, where GetParam() = (128, 64, 8) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/137
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 2048
  expected
    Which is: 1069549568
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/137, where GetParam() = (128, 64, 10) (0 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/138
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 2048
  expected
    Which is: 1056966656
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/138, where GetParam() = (128, 64, 12) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/139
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 2048
  expected
    Which is: 805308416
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/139, where GetParam() = (128, 64, 16) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/140
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 4096
  expected
    Which is: 266342400
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/140, where GetParam() = (128, 128, 8) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/141
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 4096
  expected
    Which is: 1065357312
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/141, where GetParam() = (128, 128, 10) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/142
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 4096
  expected
    Which is: 1040191488
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/142, where GetParam() = (128, 128, 12) (1 ms)
[ RUN      ] Variance/HbdVarianceTest.OneQuarterTest/143
C:\Works\AV1\SVT-AV1\test\VarianceTest.cc(468): error: Expected equality of these values:
  cal_variance(sse64_asm)
    Which is: 4096
  expected
    Which is: 536875008
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/143, where GetParam() = (128, 128, 16) (0 ms)
[----------] 432 tests from Variance/HbdVarianceTest (7780 ms total)

[----------] Global test environment tear-down
[==========] 432 tests from 1 test case ran. (7788 ms total)
[  PASSED  ] 166 tests.
[  FAILED  ] 266 tests, listed below:
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/0, where GetParam() = (4, 4, 8)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/1, where GetParam() = (4, 4, 10)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/2, where GetParam() = (4, 4, 12)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/3, where GetParam() = (4, 4, 16)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/4, where GetParam() = (4, 8, 8)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/5, where GetParam() = (4, 8, 10)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/6, where GetParam() = (4, 8, 12)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/7, where GetParam() = (4, 8, 16)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/8, where GetParam() = (4, 16, 8)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/9, where GetParam() = (4, 16, 10)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/10, where GetParam() = (4, 16, 12)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/11, where GetParam() = (4, 16, 16)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/12, where GetParam() = (4, 32, 8)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/13, where GetParam() = (4, 32, 10)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/14, where GetParam() = (4, 32, 12)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/15, where GetParam() = (4, 32, 16)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/16, where GetParam() = (4, 64, 8)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/17, where GetParam() = (4, 64, 10)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/18, where GetParam() = (4, 64, 12)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/19, where GetParam() = (4, 64, 16)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/20, where GetParam() = (4, 128, 8)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/21, where GetParam() = (4, 128, 10)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/22, where GetParam() = (4, 128, 12)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/23, where GetParam() = (4, 128, 16)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/24, where GetParam() = (8, 4, 8)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/25, where GetParam() = (8, 4, 10)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/26, where GetParam() = (8, 4, 12)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/27, where GetParam() = (8, 4, 16)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/28, where GetParam() = (8, 8, 8)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/29, where GetParam() = (8, 8, 10)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/30, where GetParam() = (8, 8, 12)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/31, where GetParam() = (8, 8, 16)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/32, where GetParam() = (8, 16, 8)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/33, where GetParam() = (8, 16, 10)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/34, where GetParam() = (8, 16, 12)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/35, where GetParam() = (8, 16, 16)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/36, where GetParam() = (8, 32, 8)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/37, where GetParam() = (8, 32, 10)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/38, where GetParam() = (8, 32, 12)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/39, where GetParam() = (8, 32, 16)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/40, where GetParam() = (8, 64, 8)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/41, where GetParam() = (8, 64, 10)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/42, where GetParam() = (8, 64, 12)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/43, where GetParam() = (8, 64, 16)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/44, where GetParam() = (8, 128, 8)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/45, where GetParam() = (8, 128, 10)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/46, where GetParam() = (8, 128, 12)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/47, where GetParam() = (8, 128, 16)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/48, where GetParam() = (16, 4, 8)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/49, where GetParam() = (16, 4, 10)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/50, where GetParam() = (16, 4, 12)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/51, where GetParam() = (16, 4, 16)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/52, where GetParam() = (16, 8, 8)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/53, where GetParam() = (16, 8, 10)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/54, where GetParam() = (16, 8, 12)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/55, where GetParam() = (16, 8, 16)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/56, where GetParam() = (16, 16, 8)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/57, where GetParam() = (16, 16, 10)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/58, where GetParam() = (16, 16, 12)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/59, where GetParam() = (16, 16, 16)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/60, where GetParam() = (16, 32, 8)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/61, where GetParam() = (16, 32, 10)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/62, where GetParam() = (16, 32, 12)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/63, where GetParam() = (16, 32, 16)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/64, where GetParam() = (16, 64, 8)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/65, where GetParam() = (16, 64, 10)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/66, where GetParam() = (16, 64, 12)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/67, where GetParam() = (16, 64, 16)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/68, where GetParam() = (16, 128, 8)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/69, where GetParam() = (16, 128, 10)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/70, where GetParam() = (16, 128, 12)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/71, where GetParam() = (16, 128, 16)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/72, where GetParam() = (32, 4, 8)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/73, where GetParam() = (32, 4, 10)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/74, where GetParam() = (32, 4, 12)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/75, where GetParam() = (32, 4, 16)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/76, where GetParam() = (32, 8, 8)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/77, where GetParam() = (32, 8, 10)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/78, where GetParam() = (32, 8, 12)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/79, where GetParam() = (32, 8, 16)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/80, where GetParam() = (32, 16, 8)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/81, where GetParam() = (32, 16, 10)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/82, where GetParam() = (32, 16, 12)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/83, where GetParam() = (32, 16, 16)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/84, where GetParam() = (32, 32, 8)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/85, where GetParam() = (32, 32, 10)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/86, where GetParam() = (32, 32, 12)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/87, where GetParam() = (32, 32, 16)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/88, where GetParam() = (32, 64, 8)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/89, where GetParam() = (32, 64, 10)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/90, where GetParam() = (32, 64, 12)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/91, where GetParam() = (32, 64, 16)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/92, where GetParam() = (32, 128, 8)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/93, where GetParam() = (32, 128, 10)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/94, where GetParam() = (32, 128, 12)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/95, where GetParam() = (32, 128, 16)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/96, where GetParam() = (64, 4, 8)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/97, where GetParam() = (64, 4, 10)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/98, where GetParam() = (64, 4, 12)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/99, where GetParam() = (64, 4, 16)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/100, where GetParam() = (64, 8, 8)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/101, where GetParam() = (64, 8, 10)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/102, where GetParam() = (64, 8, 12)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/103, where GetParam() = (64, 8, 16)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/104, where GetParam() = (64, 16, 8)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/105, where GetParam() = (64, 16, 10)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/106, where GetParam() = (64, 16, 12)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/107, where GetParam() = (64, 16, 16)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/108, where GetParam() = (64, 32, 8)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/109, where GetParam() = (64, 32, 10)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/110, where GetParam() = (64, 32, 12)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/111, where GetParam() = (64, 32, 16)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/112, where GetParam() = (64, 64, 8)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/113, where GetParam() = (64, 64, 10)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/114, where GetParam() = (64, 64, 12)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/115, where GetParam() = (64, 64, 16)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/116, where GetParam() = (64, 128, 8)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/117, where GetParam() = (64, 128, 10)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/118, where GetParam() = (64, 128, 12)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/119, where GetParam() = (64, 128, 16)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/120, where GetParam() = (128, 4, 8)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/121, where GetParam() = (128, 4, 10)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/122, where GetParam() = (128, 4, 12)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/123, where GetParam() = (128, 4, 16)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/124, where GetParam() = (128, 8, 8)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/125, where GetParam() = (128, 8, 10)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/126, where GetParam() = (128, 8, 12)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/127, where GetParam() = (128, 8, 16)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/128, where GetParam() = (128, 16, 8)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/129, where GetParam() = (128, 16, 10)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/130, where GetParam() = (128, 16, 12)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/131, where GetParam() = (128, 16, 16)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/132, where GetParam() = (128, 32, 8)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/133, where GetParam() = (128, 32, 10)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/134, where GetParam() = (128, 32, 12)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/135, where GetParam() = (128, 32, 16)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/136, where GetParam() = (128, 64, 8)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/137, where GetParam() = (128, 64, 10)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/138, where GetParam() = (128, 64, 12)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/139, where GetParam() = (128, 64, 16)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/140, where GetParam() = (128, 128, 8)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/141, where GetParam() = (128, 128, 10)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/142, where GetParam() = (128, 128, 12)
[  FAILED  ] Variance/HbdVarianceTest.ZeroTest/143, where GetParam() = (128, 128, 16)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/3, where GetParam() = (4, 4, 16)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/6, where GetParam() = (4, 8, 12)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/7, where GetParam() = (4, 8, 16)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/10, where GetParam() = (4, 16, 12)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/11, where GetParam() = (4, 16, 16)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/13, where GetParam() = (4, 32, 10)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/14, where GetParam() = (4, 32, 12)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/15, where GetParam() = (4, 32, 16)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/17, where GetParam() = (4, 64, 10)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/18, where GetParam() = (4, 64, 12)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/19, where GetParam() = (4, 64, 16)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/20, where GetParam() = (4, 128, 8)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/21, where GetParam() = (4, 128, 10)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/22, where GetParam() = (4, 128, 12)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/23, where GetParam() = (4, 128, 16)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/26, where GetParam() = (8, 4, 12)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/27, where GetParam() = (8, 4, 16)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/30, where GetParam() = (8, 8, 12)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/31, where GetParam() = (8, 8, 16)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/33, where GetParam() = (8, 16, 10)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/34, where GetParam() = (8, 16, 12)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/35, where GetParam() = (8, 16, 16)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/37, where GetParam() = (8, 32, 10)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/38, where GetParam() = (8, 32, 12)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/39, where GetParam() = (8, 32, 16)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/40, where GetParam() = (8, 64, 8)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/41, where GetParam() = (8, 64, 10)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/42, where GetParam() = (8, 64, 12)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/43, where GetParam() = (8, 64, 16)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/44, where GetParam() = (8, 128, 8)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/45, where GetParam() = (8, 128, 10)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/46, where GetParam() = (8, 128, 12)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/47, where GetParam() = (8, 128, 16)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/50, where GetParam() = (16, 4, 12)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/51, where GetParam() = (16, 4, 16)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/53, where GetParam() = (16, 8, 10)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/54, where GetParam() = (16, 8, 12)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/55, where GetParam() = (16, 8, 16)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/57, where GetParam() = (16, 16, 10)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/58, where GetParam() = (16, 16, 12)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/59, where GetParam() = (16, 16, 16)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/60, where GetParam() = (16, 32, 8)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/61, where GetParam() = (16, 32, 10)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/62, where GetParam() = (16, 32, 12)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/63, where GetParam() = (16, 32, 16)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/64, where GetParam() = (16, 64, 8)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/65, where GetParam() = (16, 64, 10)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/66, where GetParam() = (16, 64, 12)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/67, where GetParam() = (16, 64, 16)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/68, where GetParam() = (16, 128, 8)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/69, where GetParam() = (16, 128, 10)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/70, where GetParam() = (16, 128, 12)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/71, where GetParam() = (16, 128, 16)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/73, where GetParam() = (32, 4, 10)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/74, where GetParam() = (32, 4, 12)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/75, where GetParam() = (32, 4, 16)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/77, where GetParam() = (32, 8, 10)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/78, where GetParam() = (32, 8, 12)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/79, where GetParam() = (32, 8, 16)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/80, where GetParam() = (32, 16, 8)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/81, where GetParam() = (32, 16, 10)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/82, where GetParam() = (32, 16, 12)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/83, where GetParam() = (32, 16, 16)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/84, where GetParam() = (32, 32, 8)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/85, where GetParam() = (32, 32, 10)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/86, where GetParam() = (32, 32, 12)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/87, where GetParam() = (32, 32, 16)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/88, where GetParam() = (32, 64, 8)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/89, where GetParam() = (32, 64, 10)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/90, where GetParam() = (32, 64, 12)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/91, where GetParam() = (32, 64, 16)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/92, where GetParam() = (32, 128, 8)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/93, where GetParam() = (32, 128, 10)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/94, where GetParam() = (32, 128, 12)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/95, where GetParam() = (32, 128, 16)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/97, where GetParam() = (64, 4, 10)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/98, where GetParam() = (64, 4, 12)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/99, where GetParam() = (64, 4, 16)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/100, where GetParam() = (64, 8, 8)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/101, where GetParam() = (64, 8, 10)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/102, where GetParam() = (64, 8, 12)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/103, where GetParam() = (64, 8, 16)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/104, where GetParam() = (64, 16, 8)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/105, where GetParam() = (64, 16, 10)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/106, where GetParam() = (64, 16, 12)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/107, where GetParam() = (64, 16, 16)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/108, where GetParam() = (64, 32, 8)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/109, where GetParam() = (64, 32, 10)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/110, where GetParam() = (64, 32, 12)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/111, where GetParam() = (64, 32, 16)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/112, where GetParam() = (64, 64, 8)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/113, where GetParam() = (64, 64, 10)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/114, where GetParam() = (64, 64, 12)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/115, where GetParam() = (64, 64, 16)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/116, where GetParam() = (64, 128, 8)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/117, where GetParam() = (64, 128, 10)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/118, where GetParam() = (64, 128, 12)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/119, where GetParam() = (64, 128, 16)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/120, where GetParam() = (128, 4, 8)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/121, where GetParam() = (128, 4, 10)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/122, where GetParam() = (128, 4, 12)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/123, where GetParam() = (128, 4, 16)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/124, where GetParam() = (128, 8, 8)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/125, where GetParam() = (128, 8, 10)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/126, where GetParam() = (128, 8, 12)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/127, where GetParam() = (128, 8, 16)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/128, where GetParam() = (128, 16, 8)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/129, where GetParam() = (128, 16, 10)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/130, where GetParam() = (128, 16, 12)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/131, where GetParam() = (128, 16, 16)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/132, where GetParam() = (128, 32, 8)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/133, where GetParam() = (128, 32, 10)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/134, where GetParam() = (128, 32, 12)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/135, where GetParam() = (128, 32, 16)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/136, where GetParam() = (128, 64, 8)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/137, where GetParam() = (128, 64, 10)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/138, where GetParam() = (128, 64, 12)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/139, where GetParam() = (128, 64, 16)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/140, where GetParam() = (128, 128, 8)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/141, where GetParam() = (128, 128, 10)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/142, where GetParam() = (128, 128, 12)
[  FAILED  ] Variance/HbdVarianceTest.OneQuarterTest/143, where GetParam() = (128, 128, 16)

266 FAILED TESTS
~~~

Reply:
We can test the lbd version first, and we can update the test once they change the functions.